testing/matrix-sdk-integration-testing/assets/docker-compose.yml

```diff
@@ -24,20 +22,21 @@ services:
       timeout: 5s
       retries: 5
     volumes:
-      - ./data/db:/var/lib/postgresql/data
+      - db:/var/lib/postgresql/data
 
   sliding-sync-proxy:
     image: ghcr.io/matrix-org/sliding-sync:v0.99.11
     depends_on:
```

Comment:
Now that we've removed the `links`, could we add a `synapse:` line in the `depends_on` of `sliding-sync-proxy` so as to try to get synapse running *before* the sliding sync proxy? Even with it I get some races where the proxy starts before, and it may or may not be restarted later on. It's not ideal but better than not making the dependencies between the two instances explicit.

Reply:
Done